Cargo.toml

```diff
@@ -7,13 +7,17 @@ edition = "2018"
 [dependencies]
 actix = "0.9"
 bigdecimal = "0.1.0"
-diesel = { version = "1.4.4", features = ["postgres", "numeric", "serde_json"] }
-diesel-derive-enum = { version = "1", features = ["postgres"] }
+diesel = { version = "1.4.5", features = ["postgres", "numeric", "serde_json"] }
+# Using hacky diesel-derive-enum https://github.com/adwhit/diesel-derive-enum/issues/52
+diesel-derive-enum = { git = "https://github.com/khorolets/diesel-derive-enum.git", branch = "lookup-hack", features = ["postgres"] }
+futures = "0.3.5"
+itertools = "0.9.0"
 dotenv = "0.15.0"
 r2d2 = "0.8.8"
 serde_json = "1.0.55"
 tokio = { version = "0.2", features = ["sync", "time"] }
 tokio-diesel = "0.3.0"
 tracing = "0.1.13"
+tracing-subscriber = "0.2.4"
 
 near-indexer = { git = "https://github.com/nearprotocol/nearcore" }
```

Comment:
Specify commit hash here, so we don't rely on Cargo.lock and explicitly control our dependency.